### PR TITLE
Update Api docs for library 1.2.1

### PR DESCRIPTION
**Pull request by hemlok**

### CODE REVIEW

**Comment by hemlok** [comment]
Deploy preview seems to be out of date...

File: pages/page.mdx [hunk 1]
```diff
@@ -73,10 +73,6 @@ export function MyComponent() {
 <APIProperty name="PageProperties.paddingBottom" />
 <APIProperty name="PageProperties.paddingLeft" />
 
-## Events
-
-<APIMethod name="PageEvents.onChangePage()" />
-
```
Comment (jonastreub) [inline pages/page.mdx:79]: Why did you remove this?

File: pages/scroll.mdx [hunk 2]
```diff
@@ -65,11 +65,3 @@ Internally, the Scroll Component wraps all your `children` in a Frame and offset
...
 
 <APIProperty name="ScrollConfig.scrollAnimate" />
-
-## Events
-
-<APIMethod name="ScrollEvents.onScroll()" />
-
-<APIMethod name="ScrollEvents.onScrollStart()" />
-
-<APIMethod name="ScrollEvents.onScrollEnd()" />
```
Comment (jonastreub) [inline pages/scroll.mdx:75]: Same for these. Why are they removed?

File: pages/page.mdx (hunk 1 above)
Comment (hemlok) [reply pages/page.mdx:79]: It seems to have been removed? 👀

Comment (jonastreub) [reply pages/page.mdx:79]: I don't think so.

Comment (hemlok) [reply pages/page.mdx:79]: Uh oh

Comment (jonastreub) [reply pages/page.mdx:79]: It's still in the code.

**Comment by hemlok** [comment]
@jonastreub what is the correct action to take on these test errors? https://circleci.com/gh/framer/api-docs/418?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

**Comment by jonastreub** [comment]
Locally it seems to work fine

**Comment by jonastreub** [comment]
Wait `make verify-api-references` has issues indeed

**Comment by jonastreub** [comment]
Can't be found indeed.

image

**Comment by jonastreub** [comment]
I think something goes wrong in the library api generation.

**Comment by jonastreub** [comment]
Looking at the latest `framer.api.md` I can find `PageEvents` though. Not sure how exactly the docs are being generated.

**Comment by eelco** [comment]
@jonastreub the PR now is no longer missed those, right?

**Comment by jonastreub** [comment]
@hemlok reverted removing them but the API info can't be found.

**Comment by hemlok** [comment]
They're in the API export, but they're undocumented 
```
// @public (undocumented)
export interface ScrollEvents {
    // Warning: (ae-unresolved-link) The @link reference could not be resolved: Reexported declarations are not supported
    onScroll?(info: PanInfo): void;
    // Warning: (ae-unresolved-link) The @link reference could not be resolved: Reexported declarations are not supported
    onScrollEnd?(info: PanInfo): void;
    // Warning: (ae-unresolved-link) The @link reference could not be resolved: Reexported declarations are not supported
    onScrollStart?(info: PanInfo): void;
}

```

**Comment by jonastreub** [comment]
All of these events are re-exported from `./components`. Could that be the issue?

**Comment by hemlok** [comment]
> All of these events are re-exported from `./components`. Could that be the issue?

Seems unlikely cause it seems that the code is being picked up, but only that the documentation isn't coming with it. see:
```
export interface PageEvents {
    onChangePage(currentIndex: number, previousIndex: number): void;
}
```
https://github.com/framer/FramerStudio/blob/e98143de08bce2266d193987fd37465a057fc626/Source/Library/api/framer.api.md#L2331-L2332

**Comment by hemlok** [comment]
Similarly, locally, PageEvents is correctly annotated in `framer.d.ts`, it's just not being pulled in by api-extractor 🤔

**Comment by jonastreub** [comment]
This PR by introduced the scroll event warnings. That's so long ago it can't be the problem. https://github.com/framer/FramerStudio/pull/2421

**Comment by hemlok** [comment]
Nothing that seems related to this has changed in `framer.api.md` since the last release of 1.2.0 in  mid April 👀
